examples/estimateFee/go.work

```diff
@@ -0,0 +1,6 @@
+go 1.21
+
+use (
+	.
+	../..
+)
```

Comment:
In order to create this file did you run the following command: `go work init . ../..`?

Reply:
I believe this file came with one of the initial examples before we took over the project.